3_visualize/src/drought_learner_viz_components.R

```diff
@@ -76,23 +97,23 @@ bottom_bars <- function(both, droughts_df, blank_plot){
 
 
 frame_a <- function(blank_plot, streamflow_df, droughts_df,
-                    bottom_bars, canvas, inset, out_png){
+                    bottom_bars, canvas, inset, out_png, dv_tibble){
   
   main <- blank_plot +
     # Fixed threshold drought durations
     annotate("rect", # fixed threshold
              xmin = droughts_df$start[droughts_df$method == "fixed"],
              xmax = droughts_df$end[droughts_df$method == "fixed"],
              ymin = -Inf, ymax = Inf,
-             fill = "#da7d81", alpha = 0.8) +
+             fill = dv_tibble$dv_drought_fill_fixed, alpha = 0.8) +
     annotate("text", label = "Periods of\nSevere\nDrought", 
              x = as.Date("01/08/1963",'%d/%m/%Y'), hjust = 0.5,
```

Comment:
I decided to go ahead and include `focal_year` in these date calls as well, because if nothing else if we change the year, then these dates will just be that much simpler with just the "DD/MM" to manipulate. The consistency with other `as.Date()` calls is nice, too.